app/master/build_store.py

```diff
@@ -0,0 +1,50 @@
+from collections import OrderedDict
+from itertools import islice
+from typing import List
+
+from app.master.build import Build
+from app.util.exceptions import ItemNotFoundError
+
+
+class BuildStore(object):
```

Comment:
Agreed. You'll see `Class(object):` instead of `Class:` used sporadically throughout the codebase but we should standardize on the latter.